data_collection/gazette/spiders/base/imprensa_oficial.py

```diff
@@ -0,0 +1,51 @@
+from datetime import date, datetime
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class ImprensaOficialSpider(BaseGazetteSpider):
+    @staticmethod
+    def _get_next_month(current_date):
+        month = (current_date.month % 12) + 1
+        year = current_date.year + (current_date.month + 1 > 12)
+        return date(year, month, 1)
+
+    def start_requests(self):
+        current_date = self.start_date
+        end_date = date.today()
+        while current_date <= end_date:
+            year_month = current_date.strftime("%Y/%m")  # like 2015/01
+            current_date = self._get_next_month(current_date)
+            yield scrapy.Request(
+                self.url_base.format(year_month), callback=self.extract_gazette_links
+            )
+
+    def extract_gazette_links(self, response):
+        for a in response.css("h2 a"):
+            yield scrapy.Request(a.attrib["href"], dont_filter=True)
```

Comment:
**question**: Why are you setting the `dont_filter` flag? Will we have the same request used by different  gazettes?

Reply:
It's a bit odd but without `dont_filter=True` the request isn't done.